docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-video/index.scss

```diff
@@ -0,0 +1,98 @@
+@use 'uswds-core' as *;
+
+.cgdp-video {
+	@include u-margin-x(0);
+	@include u-margin-y(0);
+	&__caption {
+		@include u-padding-top(1);
+		@include u-padding-bottom(1);
+		@include u-padding-left(2);
+		@include u-margin-top(-0.5);
+		@include u-border-left(0.5, $theme-color-secondary);
+	}
+	&__media {
+		position: relative;
+		width: 100%;
+		height: 0;
+		margin-bottom: 0;
+		overflow: hidden;
+		padding-top: 0;
+		padding-bottom: 56.25%;
+		background-color: color($theme-color-accent-cool-darker);
+
+		iframe,
+		object,
+		embed,
+		video {
+			position: absolute;
+			top: 0;
+			left: 0;
+			width: 100%;
+			height: 100%;
+		}
+	}
+}
+
+// keeping these terrible names for backwards compatibility
+.video-preview--container {
+	&:hover,
+	&:focus {
+		cursor: pointer;
+
+		.play-button--bg {
+			fill: red;
+			fill-opacity: 1;
+			transition: all 100ms cubic-bezier(0, 0, 0.2, 1);
+		}
+
+    // add focus indicator for keyboard navigation
+		&:focus {
+			.video-preview--play-button {
+				outline: 0.25rem solid color($theme-color-primary-light);
+				outline-offset: 2px;
+			}
+		}
+	}
+
+	&::before {
+		z-index: 1;
+		position: absolute;
+		content: '';
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 40px;
+		background: linear-gradient(rgba(black, 0.6) 0%, rgba(black, 0.2) 60%, rgba(black, 0) 100%);
+	}
+
+	p {
+		position: absolute;
+		color: white;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+		overflow: hidden;
+		z-index: 10;
+		width: 95%;
+		margin: 0.5em 0;
+		padding: 0 15px;
+	}
+
+	// Easier than adding the previously used class back onto the element
+	// Just change the selector.
+	> img {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: auto;
+		margin-top: -9.4%;
+	}
+}
+
+.video-preview--play-button {
```

Comment:
Same as above. Also `--play-button` is a bad name, `__play-button` would be an element. I assume this follows the "keeping horrible names".